src/intervals/conversion_promotion.jl

```diff
@@ -3,18 +3,19 @@
 ## Conversion and promotion
 
 ## Default conversion to Interval, corresponds to Interval{Float64}
+# do we really need this?
```

Comment:
We *do* need this! For instance, when you calculate `x+0.1` with `x` an Interval, `0.1` is promoted from float to interval, which at some point uses `convert{Interval{Float64},0.1}` an thus yields `Interval[0.09999999999999999, 0.1]` (and not `Interval(0.1)`).